app/src/main/java/to/bitkit/ui/screens/wallets/activity/TagSelectorSheet.kt

```diff
@@ -42,13 +41,6 @@ fun TagSelectorSheet() {
         availableTags = availableTags,
         selectedTags = selectedTags,
         onTagClick = { activity.toggleTag(it) },
```

Comment:
we should also close sheet, that's what happens in RN app and it's more intuitive:

you tap the tag icon to open the sheet → you tap the tag → filter applies & sheet hides

Just needs a `app.hideSheet()` at the end of `onTagClick`